Lib/trufont/windows/fontWindow.py

```diff
@@ -678,13 +684,40 @@ def copy(self):
             packGlyphs = (
                 glyphs[index] for index in sorted(
                     self.glyphCellView.selection()))
+
+        fontHeight = font.info.ascender - font.info.descender
+        glyphSet = {n: font[n] for n in font.glyphOrder}
+        svgGlyphs = []
         pickled = []
+        x = 0
         for glyph in packGlyphs:
             pickled.append(glyph.serialize(
                 blacklist=("name", "unicodes")
             ))
+
+            if SVGPathPen is not None:
+                pen = SVGPathPen(glyphSet)
+                glyph.draw(pen)
+                width = glyph.width or glyph.bounds[2] - glyph.bounds[0]
```

Comment:
Combining marks might intentionally have zero width, but that doesn't matter.

Reply:
maybe*

Reply:
That was specifically to handle them. When copying multiple glyphs, I was putting them next to each other but marks would then overlap. I changed this and I now put the glyphs in a grid of five columns using the upm for all glyphs, it looks nicer this way.